mlflow/tracking/client.py

```diff
@@ -850,6 +852,7 @@ def print_run_info(r):
             status: FINISHED
         """
         self._tracking_client.log_param(run_id, key, value)
+        return value
```

Comment:
```suggestion
        return self._tracking_client.log_param(run_id, key, value)
```

Can we update all the backend stores to return a param value?

Reply:
Yes we can, but I didn't change them because I think mlflow.log_param and MlflowClient.log_param are user-facing, those behind them are not. Do you want every log_param method returning param value as well?

Reply:
That's a good point. Let's keep the current code.